### PR TITLE
Update get.md

### PR DESCRIPTION
**Pull request by jkodroff**

### CODE REVIEW

File: themes/default/content/docs/intro/concepts/resources/get.md [hunk 1]
```diff
@@ -7,7 +7,12 @@ menu:
     weight: 7
 ---
 
-You can use the static `get` function, which is available on all resource types, to look up an existing resource’s ID. The `get` function is different from the `import` function. The difference is that, although the resulting resource object’s state will match the live state from an existing environment, the resource will not be managed by Pulumi. A resource read with the `get` function will never be updated or deleted by Pulumi during an update.
+You can use the static `get` function, which is available on all resource types, to look up an existing resource that is not managed by Pulumi. The `get` function is different from [the `import` CLI command]({{< ref "/docs/reference/cli/pulumi_import" >}}): `pulumi import` is used bring an existing resource under management by Pulumi. `get` is used to allow the attributes of an existing resource to be used within a Pulumi program. A resource read with the `get` function will never be updated or deleted by Pulumi during an update.
```
Comment (susanev) [inline themes/default/content/docs/intro/concepts/resources/get.md:10]:
```suggestion
You can use the static `get` function, which is available on all resource types, to look up an existing resource that is not managed by Pulumi. The `get` function is different from the [`import` CLI command]({{< ref "/docs/reference/cli/pulumi_import" >}}): `pulumi import` is used to bring an existing resource under management by Pulumi. `get` is used to allow the attributes of an existing resource to be used within a Pulumi program. A resource read with the `get` function will never be updated or deleted by Pulumi during an update.
```

File: themes/default/content/docs/intro/concepts/resources/get.md [hunk 2]
```diff
@@ -7,7 +7,12 @@ menu:
...
+
+Two values are passed to the `get` function:
+
+1. The logical name Pulumi will use to refer to the resource.
+1. The physical ID that the resource has in the target cloud.
```
Comment (susanev) [inline themes/default/content/docs/intro/concepts/resources/get.md:15]:
```suggestion
Two values are passed to the `get` function:

1. Logical name Pulumi will use to refer to the resource.
1. Physical ID that the resource has in the target cloud.
```

dropping the `the`'s will make this a bit easier to scan

**Review by susanev — APPROVED** [review]

File: themes/default/content/docs/intro/concepts/resources/get.md (hunk 2 above)
Comment (jkodroff) [reply themes/default/content/docs/intro/concepts/resources/get.md:15]: Agreed, but I think it reads worse for non-scanners without "the". Would it be within our style guidelines if we bolded them?

```markdown
1. The **logical name** Pulumi will use to refer to the resource.
1. The **physical ID** that the resource has in the target cloud.
```

**Comment by jkodroff** [comment]
@susanev Having trouble verifying that my link (`relref`) to the CLI is correct. It looks like all the CLI links 404 in the generated site, even when clicking the links in the LH nav.

Please advise.

**Comment by susanev** [comment]
@jkodroff the cli links 404 in previews because the cli docs do not get built until it gets to docs. i think there's directions in a readme somewhere about how to generate the cli docs locally if you want to try that, but let me take a look at the link.

**Comment by susanev** [comment]
link looks good to me, but i honestly mess up links a lot so you might wanna have someone else verify as well.

**Comment by jkodroff** [comment]
Verified the link is correct after deployment.

**Pull request merged**